sycl/include/CL/sycl/detail/pi.h

```diff
@@ -1600,6 +1600,11 @@ __SYCL_EXPORT pi_result piextUSMGetMemAllocInfo(
     pi_context context, const void *ptr, pi_mem_info param_name,
     size_t param_value_size, void *param_value, size_t *param_value_size_ret);
 
+/// API to get Plugin/Device internal data to be directly used by
```

Comment:
> Plugin/Device

As is it is plugin-wise, not per-device. If you need finer granularity then pass `pi_device` here

Reply:
Will replace 'Plugin/Device' with 'Plugin' in the comment while rebasing & resolving conflicts.